### PR TITLE
Fix #165624 check-host.net

### PR DESCRIPTION
**Pull request by xglossyntx**
# Creating the pull request

>This rule still needs to be updated `check-host.net#?#div + div[class*=" "]:has(> div[style] > div > div:contains(/^ ?[×✕✖X☓χx⨯✗\+\＋\﹢хХΧẊẋ] ?$/))`

### To avoid invalid pull requests, please check and confirm following terms

- [x] This is not an ad/bug report;
- [x] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [x] I have performed a self-review of my own changes;
- [x] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [x] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

### Enter the issue address

#165624

### Terms

- [x] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

**Comment by xglossyntx** [comment]
Strange it really does not want to let me remove that

File: BaseFilter/sections/specific.txt [hunk 1]
```diff
@@ -16848,11 +16848,11 @@ outlook.live.com#?##app > div[class] > div[class] > div[class] > div[class]:not(
...
 ! check-host.net
-check-host.net##div[class*="place-self-"]
-check-host.net##div[class*="relative "]
-/^https:\/\/check-host\.net\/images\/[a-z]{2,11}[0-9]{1,3}-?[0-9]+[a-z]+(_|[0-9]?[a-z-])+(s?ale|en|ru)?\.png/$domain=check-host.net
+check-host.net##div[class*="-self-"]
+check-host.net##a[href*="incogni.io"]
+/^https:\/\/check-host\.net\/rimages\/[a-z]{2,15}[0-9]{1,3}-?[0-9]+[a-z]+(_|[0-9]?[a-z-])+(s?ale|en|ru)?\.png/$domain=check-host.net
 check-host.net#?#div + div[class*=" "]:has(> div[style] > div > div:contains(/^ ?[×✕✖X☓χx⨯✗\+\＋\﹢хХΧẊẋ] ?$/))
-check-host.net#?#a[href]:contains(/^(на|visit )?([Aa][eé]za|100[%％﹪⁒]| )/)
+check-host.net#?#a[href]:contains(/[Aa][eé]za|100[%％﹪⁒]| |Incogni/):upward(1)
```
Comment (zloyden) [inline BaseFilter/sections/specific.txt:16855]: Actually, the rule blocks the useful elements. I've updated them directly.

**Comment by xglossyntx** [comment]
What was it blocking that it should not have been?

**Comment by xglossyntx** [comment]
Never mind, I see it blocks the "Permanent link to this check report" button.

**Comment by zloyden** [comment]
<details><summary>Screenshot:</summary>

image

</details><br/>

**Pull request closed without merging**